src/threechances.cpp

```diff
@@ -119,8 +119,8 @@ void ThreeChances::initialize(HWND hwnd) {
 	//if (!slug.initialize(this, TILE_SIZE, TILE_SIZE, SLUG_COLS, &slugTexture, SLUG_DATA))
 	//	throw(GameError(gameErrorNS::FATAL_ERROR, "Error initialising slug monster"));
 
-	//if (!sword.initialize(this, 112, 112, 4, &swordTexture))
-	//	throw(GameError(gameErrorNS::FATAL_ERROR, "Error initializing sword"));
+	if (!sword.initialize(this, 128, 128, 4, &swordTexture))
```

Comment:
Use constants for these values
e.g. `SWORD_WIDTH`, `SWORD_HEIGHT`, `SWORD_COLS`